psiminer-core/src/test/resources/data/java/JavaEmptyMethods.java

```diff
@@ -0,0 +1,44 @@
+// abstract methods
+interface Interface {
+    void interfaceMethod();
+}
+
+abstract class InternetProvider {
```

Comment:
Since we doesn't use internet assosiated things in tests it's better to rename this too